### PR TITLE
Remove CHT tutorial cases

### PR DESCRIPTION
**Pull request by DavidSCN**
As they are now part of the tutorials, related to precice/tutorials#125.

### CODE REVIEW

**Review by MakisH — COMMENTED** [review]
We should also update the `README.md` of the repository regarding this.

So, we keep the Fluid-Fluid case here for now, and deal with it in another PR. We can ignore them in the `README.md`.

**Comment by DavidSCN** [comment]
With 'in another PR' you mean after finishing the restructuring, right? As it is currently not part of the restructuring.

**Comment by DavidSCN** [comment]
I updated the README and removed the 'Testing phase' slogan as well. Feel free to review again and or merge.

File: README.md [hunk 1]
```diff
@@ -12,13 +12,13 @@ The fluid-structure interaction module was developed in close collaboration betw
 
 The fluid-fluid coupling module is being developed by Gerasimos Chourdakis, in the context of his dissertation.
 
-The adapter is [easily extensible](https://github.com/precice/openfoam-adapter/wiki/How-to-extend-the-adapter).
+The adapter is [easily extensible](adapter-openfoam-extend.html).
 
 ## Start here
 
-Our [wiki](https://github.com/precice/openfoam-adapter/wiki) will help you start. 
+Our [wiki](adapter-openfoam-overview.html) will help you start.
```
Comment (MakisH) [inline README.md:19]:
```suggestion
Our [documentation](adapter-openfoam-overview.html) will help you start.
```

File: README.md [hunk 2]
```diff
@@ -12,13 +12,13 @@ The fluid-structure interaction module was developed in close collaboration betw
...
 
-You may find OpenFOAM-only tutorials here, in order to test your installation, or cases involving other adapters in the [precice/tutorials](https://github.com/precice/tutorials) repository. Step-by-step instructions for them are provided in the [preCICE wiki](https://github.com/precice/precice/wiki).
+You may find OpenFOAM-only tutorials here, in order to test your installation, or cases involving other adapters in the [precice/tutorials](https://github.com/precice/tutorials) repository. Instructions for them are provided in the [tutorial documentation](tutorials.html).
```
Comment (MakisH) [inline README.md:21]:
```suggestion
You can find examples in the [precice/tutorials](https://github.com/precice/tutorials) repository. Instructions for them are provided in the [tutorials documentation](tutorials.html).
```

**Review by MakisH — APPROVED** [review]

**Pull request merged**